src/core/server/saved_objects/migrationsv2/actions/index.ts

```diff
@@ -184,10 +184,10 @@ export const removeWriteBlock = (
  * yellow at any point in the future. So ultimately data-redundancy is up to
  * users to maintain.
  */
-const waitForIndexStatusYellow = (
+export const waitForIndexStatusYellow = (
```

Comment:
There isn't an integration test for `waitForIndexStatusYellow` but we previously indirectly tested it through the `cloneIndex` and `createIndex` methods, but perhaps it's better to add separate test now that this method is public?

Reply:
done